qmt/geometry/freecad/auxiliary.py

```diff
@@ -48,15 +48,20 @@ def silent_stdout():
     '''Suppress standard output.'''
     sys.stdout.flush()
     stored_py = sys.stdout
-    stored_fileno = os.dup(sys.stdout.fileno())
+    stored_fileno = None
```

Comment:
this is so that jupyter notebook still works because it doesn't support `fileno()`. See https://stackoverflow.com/questions/37882022/error-iostream-has-no-fileno-sumo